dev/build/windows/makecoq_mingw.sh

```diff
@@ -959,6 +959,10 @@ function make_lablgtk {
     "$TARGET_ARCH-strip.exe" --strip-unneeded src/dlllablgtk2.dll
 
     log2 make world
+
+    # lablgtk does not escape FINDLIBDIR path, which can contain backslashes
+    sed -i "s|^FINDLIBDIR=.*|FINDLIBDIR=$PREFIXOCAML/libocaml/site-lib|" config.make
+
```

Comment:
Can you explain under which circumstances FINDLIBDIR contains backslashes? 
FINDLIBDIR should be in what I call MFMT.
See the docs in MakeCoq_MinGW.bat:

REM WFMT = windows format (C:\..)          Used in this batch file.
REM CFMT = cygwin format (\cygdrive\c\..)  Used for Cygwin PATH varible, which is : separated, so C: doesn't work.
REM MFMT = MinGW format (C:/...)           Used for the build, because \\ requires escaping. Mingw can handle \ and /.

Reply:
This path is built by lablgtk calling `ocamlfind`, so we don't have control over the format.